{"uuid": "1d425f8e-ccce-4454-a966-983c2ebc72be", "children": ["ed752465-6cf5-4256-b34b-0c1d32b527bf"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/e2e/basic/conftest.py\", line 80, in setup_profiles\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/fixtures_4x.py\", line 111, in setup_profiles\n    instantiate_profile_obj.check_and_set_ssid_sec(\"2G\", profile_data[\"ssid\"][mode][0][\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 677, in check_and_set_ssid_sec\n    self.get_all_ssid_detail(radio=radio)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 724, in get_all_ssid_detail\n    ssid_details = self.run_generic_cli_command(cmd)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 180, in run_generic_cli_command\n    output = os.popen(cmd).read()\n"}, "start": 1702295770900, "stop": 1702295926107}], "start": 1702295770900, "stop": 1702295926587}